dexcom/test/calibration.go

```diff
@@ -42,14 +44,22 @@ func CloneCalibrations(datum *dexcom.Calibrations) *dexcom.Calibrations {
 
 func RandomCalibration() *dexcom.Calibration {
 	datum := dexcom.NewCalibration()
+	datum.ID = pointer.FromString(test.RandomString())
 	datum.SystemTime = RandomSystemTime()
 	datum.DisplayTime = RandomDisplayTime()
 	datum.Unit = pointer.FromString(test.RandomStringFromArray(dexcom.CalibrationUnits()))
 	switch *datum.Unit {
 	case dexcom.CalibrationUnitMgdL:
 		datum.Value = pointer.FromFloat64(test.RandomFloat64FromRange(dexcom.CalibrationValueMgdLMinimum, dexcom.CalibrationValueMgdLMaximum))
+	case dexcom.CalibrationUnitMmolL:
```

Comment:
nitpick: since `case dexcom.CalibrationUnitMmolL:` and `case dexcom.CalibrationUnitUnknown:` assign the same values, could just combine the two into `case dexcom.CalibrationUnitMmolL, dexcom.CalibrationUnitUnknown:` - unless there's some future logic that the two will additionally do.